src/components/shared/patientDataForm/PatientDataForm.tsx

```diff
@@ -17,14 +18,16 @@ const PatientDataForm: FunctionComponent<TProps> = ({
   submitButtonLabel,
 }) => {
   const validationSchema = object({
-    name: string().required("This field is required"),
+    firstName: string().required("This field is required"),
     //TODO: write schema
   });
 
   const formik = useFormik({
     initialValues,
     validationSchema,
-    onSubmit: handleSubmit,
+    onSubmit: (values: PatientDTO) => {
+      handleSubmit(values);
```

Comment:
Pass handleSubmit to useFormik directly and change the property name (should be onSubmit) for keeping nomenclature coherence.